sweepbatcher/sweep_batch.go

```diff
@@ -715,10 +715,11 @@ func (b *batch) publishBatch(ctx context.Context) (btcutil.Amount, error) {
 
 	weightEstimate.AddP2TROutput()
 
-	fee = b.rbfCache.FeeRate.FeeForWeight(weightEstimate.Weight())
+	weight := weightEstimate.Weight()
+	fee0 := b.rbfCache.FeeRate.FeeForWeight(weight)
```

Comment:
nit: instead of `fee0` and `fee` we could call them `feeForWeight` and `clampedFee`.

Reply:
I renamed to `feeForWeight` and `fee`.